tools/pontoon/ui.md

```diff
@@ -20,7 +20,7 @@ To navigate to another project, click on each item to open the drop-down menu an
 
 Clicking `All` or `Current` in the drop-down menu might navigate away from the translation workspace:
 * In the `Locales` section, `All Teams` links to the *Teams page*, while `Current Team` links to the *Team page* for the current locale.
-* In the `Project` section, `All Projects` links to the *Projects page*, while `Current Project` links to the *Project page* for the current project.
```

Comment:
Since I cannot comment on unchanged lines: "In the `Locales` section" also needs an update.

All Teams -> Teams Dashboard
Current Team -> Current Team Dashboard

Reply:
Thanks, updated.